client/src/Components/ProductCard.tsx

```diff
@@ -49,17 +47,9 @@ export default function ProductCard(
           </div>
         </div>
       </div>
-      <div
-        className="center"
-        style={{
-          borderLeft: "2px solid black",
-          height: "80%",
-          marginTop: "auto",
-          marginBottom: "auto",
-        }}
-      ></div>
+      <div className="center"></div>
       <div className="right">
-        <h2>flavour: {props.name}</h2>
+        <h2 className="title" style={{fontSize: "2rem"}}>flavor: {props.name}</h2>
         <p>
           <strong>description</strong>: <span>{props.description}</span>
```

Comment:
Don't recall having a field to store these in the DB now, might need to remove for now.

Reply:
Which one, description?